Sources/SmileID/Classes/BiometricKYC/OrchestratedBiometricKycViewModel.swift

```diff
@@ -93,23 +102,20 @@ internal class OrchestratedBiometricKycViewModel: ObservableObject {
                 let prepUploadResponse = try await SmileID.api.prepUpload(
                     request: prepUploadRequest
                 ).async()
-                let _ = try await SmileID.api.upload(
+                _ = try await SmileID.api.upload(
                     zip: zip,
                     to: prepUploadResponse.uploadUrl
                 ).async()
-                let jobStatusRequest = JobStatusRequest(
-                    userId: userId,
-                    jobId: jobId,
-                    includeImageLinks: false,
-                    includeHistory: false,
-                    timestamp: authResponse.timestamp,
-                    signature: authResponse.signature
-                )
-                jobStatusResponse = try await SmileID.api.getJobStatus(
-                    request: jobStatusRequest
-                ).async()
+                didSubmitBiometricJob = true
+                do {
+                    try LocalStorage.moveToSubmittedJobs(jobId: self.jobId)
+                } catch {
+                    print("Error moving job to submitted directory: \(error)")
+                    self.error = error
```

Comment:
If we're setting this here, we should show the error right?

```suggestion
                    self.error = error
                    DispatchQueue.main.async { self.step = .processing(.error) }
                    return
```